src/components/Graph/Graph.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import {getTopology, getYdbPlanNodeShape} from '@gravity-ui/paranoid';
+import type {Data, GraphNode, Options, Shapes} from '@gravity-ui/paranoid';
+
+interface GraphProps<T> {
+    data: Data<T>;
+    opts?: Options;
+    shapes?: Shapes;
+}
+
+export function Graph<T>(props: GraphProps<T>) {
+    const containerRef = React.useRef<HTMLDivElement>(null);
+    const containerId = React.useId();
+
+    const {data, opts, shapes} = props;
+
+    React.useEffect(() => {
+        const graphRoot = containerRef.current;
+        if (!graphRoot) {
+            return undefined;
+        }
+
+        graphRoot.innerHTML = '';
+
+        const topology = getTopology(graphRoot.id, data, opts, shapes);
+        topology.render();
+        return () => {
+            topology.destroy();
+        };
+    }, [data, opts, shapes]);
+
+    return <div id={containerId} ref={containerRef} style={{height: '100vh'}} />;
```

Comment:
There is much more info now, `100vh` isn't enough for expanded schema. Could we make `height` dynamic?

<img width="371" alt="Screenshot 2024-07-29 at 18 00 31" src="https://github.com/user-attachments/assets/48848454-3c74-468c-8559-a20fd7be1748">

Reply:
I don't see an easy way to do this. 100vh was chosen as a height greater than the maximum height of the container. The same should be done with the width. If schema does not fit into visible area, then you can move it with the mouse

Reply:
I didn't know, that it can be moved by mouse. In this case, it's not a big problem